index.html

```diff
@@ -43,7 +43,154 @@ <h1>I'm Shahira <br> Glad to see <br>you!</h1>
           </ul>
         </section>
 
-    </main>
+        <div class="card-works">
```

Comment:
- [ ] I suggest you use a `<section>` tag for this container. The `<section>` element is a semantic element; it provides meaning to both user agents and humans about the enclosed content — specifically, a section of the document. The sub-containers in this container can be just `divs`.

Reply:
- [ ] As the requirements suggest, In order to lay out the elements in the `"Works"` section you should use CSS Grid and Flexbox combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card.